README.md

```diff
@@ -20,7 +20,7 @@ And that is all!
 
 Create new project on Shopsys Framework easily from this package following one of the installation guides:
 
-* [Installation via Docker (recommended)](https://github.com/shopsys/shopsys/blob/master/docs/installation/installation-guide.md)
+* [Installation via Docker (recommended)](https://github.com/shopsys/shopsys/blob/master/docs/installation/installation-using-docker.md)
```

Comment:
I think we might want this in master because the old link returns 404

Reply:
Makes sense, fixed in d2be104b8ac77fa26776d2601d697f23f6e3f0e5

Please, next time use a suggestion, it would save me some time 🙂